### PR TITLE
not ok option to alert list, fixes: #9754

### PR DESCRIPTION
**Pull request by Ijin08**

### CODE REVIEW

File: pkg/services/sqlstore/alert.go [hunk 1]
```diff
@@ -94,7 +94,12 @@ func HandleAlertsQuery(query *m.GetAlertsQuery) error {
 			if i > 0 {
 				sql.WriteString(" OR ")
 			}
-			sql.WriteString("state = ? ")
+			if strings.HasPrefix(v, "not") {
+				sql.WriteString("state <> ? ")
+				v = v[4:]
```
Comment (bergquist) [inline pkg/services/sqlstore/alert.go]: I think its more explicit to use strings.Replace Ex
```go
v = strings.Replace(v, "not", "", 1)
```

Comment (torkelo) [reply pkg/services/sqlstore/alert.go]: but that is less correct, if we ever add a state with the char pattern "not" inside it. 

Maybe adding a comment is enough "// remove not_ prefix" , go does not have a strings.Substring, as that is done via slice syntax like above

Comment (bergquist) [reply pkg/services/sqlstore/alert.go]: totally right

A better solution would be
```go
v = strings.TrimPrefix(v, "not")
```
https://golang.org/src/strings/strings.go?s=21287:21327#L806

Comment (torkelo) [reply pkg/services/sqlstore/alert.go]: agree! will fix and merge

**Pull request merged**